case_mapping/uco/observable.py

```diff
@@ -1342,42 +1408,61 @@ def __init__(self, path: str) -> None:
         self._str_vars(**{"uco-observable:path": path})
 
 
-class FacetEvent(FacetEntity):
+class EventRecordFacet(FacetEntity):
     def __init__(
         self,
-        event_type=None,
-        event_text=None,
-        event_id=None,
-        cyber_action=None,
-        computer_name=None,
-        created_time=None,
-        start_time=None,
+        account: Union[None, ObjectEntity] = None,
+        application: Union[None, ObjectEntity] = None,
+        cyber_action: Union[None, ObjectEntity] = None,
         end_time=None,
+        event_record_device: Union[None, ObjectEntity] = None,
+        event_record_id: Optional[str] = None,
+        event_record_raw: Optional[str] = None,
+        event_record_service_name: Optional[str] = None,
+        event_record_text: Optional[str] = None,
+        event_type: Optional[str] = None,
+        observable_created_time=None,
```

Comment:
Please assign typing to this.  It's a good chance to double-check against the cardinality restrictions in the ontology.